scripts/lib/utils

```diff
@@ -65,14 +65,11 @@ function _git() {
 }
 
 function clone_repo() {
-    if [[ -d "projects/${project}" ]]; then
-        _git fetch -f --tags
-    else
-        mkdir -p projects
-        git clone "https://github.com/${ORG}/${project}" "projects/${project}"
-        _git config advice.detachedHead false
-    fi
+    [[ -d "projects/${project}" ]] && rm -rf "projects/${project}"
 
+    mkdir -p projects
```

Comment:
Fun fact: this isn’t necessary, `git clone` will take care of creating any missing directories ;-).

Reply:
Ah nice to know, I'll send a cleanup to remove this